bridge_ui/src/components/BridgeWidget/TransactionTableCompact.tsx

```diff
@@ -44,14 +44,13 @@ const useStyles = makeStyles(() => ({
     justifyContent: 'center',
     width: 18,
     height: 18,
-    borderRadius: '50%',
-    marginRight: 8
```

Comment:
I changed the position of the icon cause it looked misaligned

| before | after |
| ----- | ----- |
| <img width="350" height="1432" alt="image" src="https://github.com/user-attachments/assets/f7d34670-62d0-4d52-8328-0a4bdd1de129" /> | <img width="350" height="698" alt="image" src="https://github.com/user-attachments/assets/ad1893b0-a9aa-4086-b784-e358125b7833" /> |